s2fft/precompute_transforms/construct.py

```diff
@@ -44,6 +47,10 @@ def spin_spherical_kernel(
 
         using_torch (bool, optional): Desired frontend functionality. Defaults to False.
 
+        mode (str, optional): Recursion to adopt. Supported recursion schemes include
+            {"auto", "price_mcewen", "risbo"}. Defaults to "auto" which will detect the
```

Comment:
I would perhaps made the settings `price-mcewen`, i.e. hypenate.  That more closely follows how this would be referenced in text.